spec/models/location_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
```

Comment:
These technically look like feature tests since you are using the `scenario` keyword, so they should be pushed into the `features` folder and not the `model` folder.